stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/header/MessagesHeaderViewBinding.kt

```diff
@@ -33,4 +34,12 @@ public fun ChannelHeaderViewModel.bindView(view: MessagesHeaderView, lifecycle:
         }
     }
     typingUsers.observe(lifecycle, view::showTypingStateLabel)
+
+    activeThread.observe(lifecycle) { message ->
+        if (message != null) {
+            view.setThreadTitle(view.context.getString(R.string.stream_ui_title_thread_reply))
```

Comment:
Seems a bit odd that the binding contains the specific string to set. Could we push that into the View itself, and just call `setThreadMode()` or something like that here?

Reply:
🤦  Yep. Translation would be totally wrong this way

Reply:
But ChatFragment is actually in the sample app... so no problem with translations, ops.

Reply:
Shouldn't the View still know how to display "Thread reply" on its own though?

Reply:
If we push this tot the View, then it will have business logic. I believe it is better to simple set the title and the subtitle so the users of the library could decide the logic they should use. What do you think?

Reply:
Now there's the normal mode and thread mode. The title is configured in the XML